pages/docs/community/styleguide/inclusive-language.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Inclusive Language style guide
+description: This style guide outlines the guideline for using inclusive languages in documentation.
+---
+
+# Inclusive Language
+
+While writing documentation we are indirectly reaching out to thousands of people all around the globe. 
+Thus, we must ensure that our diverse audience can connect with our information. 
+
+### 1. Culturally Inclusive Language
+
+Some phrases and words are popularly used only in specific regions of the world, thus our audience from other parts 
+of the world might need help to relate to the same. Hence, we should try to avoid using such phrases in our documentation. 
+Some examples of Region Specific Language are -
+- “It’s not rocket science” is commonly used in the USA, and non-US people might not be able to relate to it properly.
+- Phrases written in regional languages instead of English.
+- “It’s not a piece of cake” is commonly used in the USA, and non-US people might not be able to relate to it properly.
+
+### 2. Gender Neutral Language - 
+
+Some phrases and words are targeting men or men/women specifically. This might result in other gender groups feeling 
+left out while reading our documentation. 
+Some examples of Gender Neutral Language are -
+- Using “they/them” instead of “his/him” or “she/her”.
+- Using “Hello everyone” instead of “Hello guys”.
+- Using “Chairperson” instead of “Chairman” or “Chairwoman”.
+
+### 3. People First Language - 
+
+The placement of the word “People” matters a lot when it comes to writing documentation. 
+Using the word “People” in the beginning makes it more pleasant as the problem is not displayed in the beginning.
+Some examples of Gender Neutral Language are -
+- Using “People with Mental Instability” instead of “Mentally Unstable People”.
+- Using “People with Audio Impairment” instead of “Deaf People”.
+
+### 4. Slang Free Language -
+
+While framing the documentation, we must ensure that we are not including any vulgar language even if those are 
+included indirectly in our work. Often we don’t realize that slang words are included within phrases which have 
+don’t have collaborative slang meanings. 
+Some examples include -
+- Using “archaic” or “stupid” instead of “moron”.
```

Comment:
```suggestion
- Using “archaic” or “stupid” instead of “moron”.
```

Both terms “stupid” and "moron" are inappropriate examples historically used as diagnostic categories for individuals with mild to moderate intellectual disability; they are now considered outdated and offensive.

Using derogatory terms such as "moron," "idiot," or "retard" to refer to individuals with intellectual disabilities is stigmatizing and hurtful. It reinforces negative stereotypes and contributes to marginalization and discrimination.

**I would not use these examples under `Slang`, we must identify kind and respectful slang examples.**